gallery/2021/portrait/portrait.pug

```diff
@@ -0,0 +1,150 @@
+include mixins/active-link
```

Comment:
If you use extends /base, you won't have to rewrite the same code over and over.